### ISSUE TITLE
23899170: Xcode UI Testing fails to type text: Neither element nor any descendant has keyboard focus

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#6976]
#### Description

Summary:
Given UIViewController subclass with two UITextFields I can not type text into second text UITextField with error: UI Testing Failure - Neither element nor any descendant has keyboard focus.

Steps to Reproduce:
1. Open attached Xcode project
2. Run tests for TextEditing scheme

Expected Results:
Test should succeed

Actual Results:
Test fails with error: UI Testing Failure - Neither element nor any descendant has keyboard focus.

Version:
Xcode 7.2 (7C68)

Notes:

Configuration:
The bug occurs only in iOS 9.2 Simulator. iOS 9.0 and 9.1 are not affected.

Attachments:
'TextEditing.zip' was successfully uploaded.
## 

Product Version: 7.2 (7C68)
Created: 2015-12-15T12:22:16.258380
Originated: 2015-12-15T15:19:00
Open Radar Link: http://www.openradar.me/23899170
